<launch>
    <!-- Node to perform DOPE Pose Estimation -->
    <node name="deep_pose_estimation" pkg="dope_node" type="inference.py" output="screen">
        <!-- Parameters for the DOPE Node -->
        <param name="config" value="/home/bruno/ros_ws/ws_personal/src/dope_node/src/configs/config_pose.yaml" />
        <param name="camera_info" value="/home/bruno/ros_ws/ws_personal/src/dope_node/src/configs/camera_info.yaml" />
        <param name="weights" value="/home/bruno/ros_ws/ws_personal/src/dope_node/src/weights/coffee_box.pth"/>
        <param name="object_class" value="coffee_box" />
        <!--<param name="weights" value="/home/bruno/ros_ws/ws_personal/src/dope_node/src/weights/car_lock_black.pth"/>-->
        <!--<param name="object_class" value="car_lock_black" />-->
        <param name="image_topic" value="/usb_cam_node/image_raw" />
    </node>
</launch>

